app-backend/common/src/main/scala/com/rasterfoundry/common/BacksplashConnectionFactory.scala

```diff
@@ -1,10 +1,12 @@
 package com.rasterfoundry.common
 
-import net.spy.memcached.{ClientMode, DefaultConnectionFactory}
+import net.spy.memcached.{ClientMode, DefaultConnectionFactory, FailureMode}
 
 class BacksplashConnectionFactory extends DefaultConnectionFactory() {
   override def getClientMode: ClientMode = Config.memcached.clientMode
 
   override def getOperationTimeout: Long =
     Config.memcached.timeout
+
+  override def getFailureMode: FailureMode = FailureMode.Cancel
```

Comment:
Calling this out as well - the default is to redistribute -- but we only ever have a single node up (this could change); however, even with only a single node the client seems to infinitely try reconnecting rather than just failing. Similar to the db timeout change this is an attempt to try and surface real issues distinguishable from generic timeouts in the healthcheck.